cmaes/__init__.py

```diff
@@ -6,4 +6,9 @@
 from ._dxnesic import DXNESIC  # NOQA
 from ._catcma import CatCMA  # NOQA
 
+try:
+    from ._safe_cma import SafeCMA  # NOQA
+except ImportError:
+    pass  # Implementation of Safe CMA-ES requires scipy, gpytorch, and torch
+
```

Comment:
How about simply removing `SafeCMA` and renaming `cmaes/_safe_cma.py` to `cmaes/safe_cma.py`?

```suggestion
```

Reply:
Thanks for the great suggestion! It looks good to me.

@kento031 
Could you make this modification if it’s okay?